File: Recursion/Processing/ej11.java
float x, y; 
float a;    
float nx, ny;

void setup(){
  size(400, 400);
  background(255);
  
  x = 0;
  y = 350;
  nx = 50;
  ny = 400;
  a = 0; 

  for (int i = 0; i < 10; i++) {
    forward(50);
    turn(90);
  }
}

void drawlines(){
    line(x,y, nx, ny);
}

File: Recursion/Processing/ej11.js
    function setup() {
    createCanvas(400, 400);
  }
  function ej11(unpoco){
    unpoco += 10;
    line(0, 350-unpoco, 50+unpoco, 350-unpoco);
    line(50+unpoco, 350-unpoco, 50+unpoco, 400+unpoco);
    ej11(unpoco);
  }
  
  function draw() {
    background(200);
    let unpoco = 0;
    ej11(unpoco);
  }